uxarray/io/_exodus.py

```diff
@@ -50,58 +50,54 @@ def _read_exodus(ext_ds, grid_var_names):
         elif key == "coord":
             ds.Mesh2.attrs['topology_dimension'] = INT_DTYPE(
                 ext_ds.dims['num_dim'])
-            ds["Mesh2_node_x"] = xr.DataArray(
-                data=ext_ds.coord[0],
-                dims=["nMesh2_node"],
-                attrs={
-                    "standard_name": "longitude",
-                    "long_name": "longitude of mesh nodes",
-                    "units": "degrees_east",
-                })
-            ds["Mesh2_node_y"] = xr.DataArray(
-                data=ext_ds.coord[1],
-                dims=["nMesh2_node"],
-                attrs={
-                    "standard_name": "latitude",
-                    "long_name": "latitude of mesh nodes",
-                    "units": "degrees_north",
-                })
+            ds["Mesh2_node_x"] = xr.DataArray(data=ext_ds.coord[0],
+                                              dims=["nMesh2_node"],
+                                              attrs={
+                                                  "standard_name": "x",
+                                                  "long_name": "cartesian x",
```

Comment:
This will be tackled in #413, since it requires a bit of re-writing of existing functions. However, you are correct.

What's happening right now is that the Exodus reader assumes the coordinates will be Cartesian (x, y, z) and uses `uxarray.grid.coordinates._populate_lonlat_coord` to populate the lat/lon coordinates, however the function is written to expect `Mesh2_node_x` `Mesh2_node_y` and `Mesh2_node_z` to be populated with Cartesian coordinates.

Reply:
Thank you very much for your explanations.